### PR TITLE
Feature/library

### PR DESCRIPTION
**Pull request by GzimAsani**
> This project was created with javascript / html / css / bootstrap
> This project allows users to add their books
> Allows users to remove books
> Allows users to toggle the read status of a book

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+- Bootstrap
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+- Clone this repository
+ > `git clone https://github.com/Div685/JS-Library.git`
+- Navigate to the Repo Folder
```
Comment (whiz25) [inline README.md]: -How to navigate? Be explicit in your instruction. Add the command for navigating.

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+- Use your favorite browser the index.html file (located at the root)
```
Comment (whiz25) [inline README.md]: - This is not clear `Use your favorite browser the index.html file (located at the root)` or `Use your favorite browser to open the index.html file (located at the root)`? Make this clear.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Well done team @GzimAsani  and @Div685 . You managed to complete most of the requirements for this PR. 
Each book has a constructor for creating an instance of a book to be stored in an array as requested and the README file is well written.
However, we can't merge this PR yet. Let's make some changes.
- [ ] Add `.vscode` folder to `.gitignore`. This folder has `settings` and `configurations` about your app. It might not be a good idea to make it public.
- [ ] Read my comments for more suggestions


### Martin Afani

File: README.md (hunk 2 above)
Comment (GzimAsani) [reply README.md]: Thank you for your review, we have updated our README file according to your review!

**Review by SmithVyne — APPROVED** [review]
Approved 👍 👍 
Hello team. Congratulations on making all the changes highlighted by the previous TSE. Your project is approved and you can now merge this branch.

**Pull request merged**